src/decorators/IntField.ts

```diff
@@ -0,0 +1,30 @@
+import { Field, Int } from 'type-graphql';
+import { Column } from 'typeorm';
+
+import { composeMethodDecorators, MethodDecoratorFactory } from '../utils';
+
+interface IntFieldOptions {
+  nullable?: boolean;
+  unique?: boolean;
+}
+
+export function IntField(args: IntFieldOptions = {}): any {
+  const nullableOption = args.nullable === true ? { nullable: true } : {};
+  const uniqueOption = args.unique ? { unique: true } : {};
```

Comment:
We probably don't need this to be `unique`able, right?

Reply:
probably. I'll update